modules/main.js

```diff
@@ -0,0 +1,94 @@
+export default class Book {
+  constructor() {
+    this.section = document.getElementById('book-section');
+    this.form = document.querySelector('form');
+    this.titleInput = document.getElementById('title-input');
+    this.authorInput = document.getElementById('author-input');
+    this.DATA = JSON.parse(localStorage.getItem('data')) || [];
+    this.counter = 0;
+
+    this.list = document.querySelector('#list');
+    this.booksWrapper = document.querySelector('#books-wrapper');
+
+    this.addNew = document.querySelector('#addNew');
+    this.form = document.querySelector('#form');
+
+    this.contact = document.querySelector('#contact');
+    this.contactPage = document.querySelector('#contact-page');
+
+    if (this.DATA.length > 0) {
+      this.show();
+    }
+  }
+
+  add(author, title) {
+    this.DATA = this.DATA.concat({ author, title });
+    localStorage.setItem('data', JSON.stringify(this.DATA));
+  }
```

Comment:
- [ ] **Optional:** This function can also be written as an arrow function if you want.
```
functionName = (param, param2) => {
    // code
  };
```